src/server/handlers/store.js

```diff
@@ -27,7 +27,10 @@ export const getSnapDetails = async (req, res) => {
     });
     const json = await response.json();
 
-    res.status(response.status).send(json);
+    res.status(response.status).send({
+      status: json.error_list ? 'error' : 'success',
```

Comment:
We need to return `status: 'success'` from API for `callApi` middleware to correctly treat it as a success.